pkgs/development/python-modules/mne-python/disable-pytest-cov.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/pyproject.toml b/pyproject.toml
+index 8e7f70310..598d2368e 100644
+--- a/pyproject.toml
++++ b/pyproject.toml
+@@ -40,7 +40,7 @@ ignore-decorators = [
+ ]
+ 
+ [tool.pytest.ini_options]
+-addopts = """--durations=20 --doctest-modules -ra --cov-report= --tb=short \
```

Comment:
since this is a single line change it might be more convenient to use `substituteInPlace` or `sed` (although I think this is fine).